package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "microsummer",
+  "version": "1.0.0",
+  "description": "microservices via Node",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack — entry ./app/public/js/index.js - output-filename bundle.min.js - output-path ./app/public/js",
```

Comment:
Why you leave webpack here?

Reply:
Removed everything unused from package.json